internal/pkg_web/pkg_web.bzl

```diff
@@ -48,12 +51,15 @@ def move_files(output_name, files, action_factory, var, assembler, root_paths):
     www_dir = action_factory.declare_directory(output_name)
     args = action_factory.args()
     args.add(www_dir.path)
+    args.add(version_file.path)
```

Comment:
I think this needs some conditional logic because the version_file isn't always present?

I think we need to detect use of `--stamp` like pkg_npm does `stamp = ctx.attr.node_context_data[NodeContextInfo].stamp`

Reply:
as far as I know, the version file is always present but the content changes regarding to the `--workspace_status_command`. Since we have logic in bundler which checks for the keys, well be okay I guess.

Reply:
As far as I understand the rule has to obey --stamp which is stored in the node_context helper rule. I should make this work that way.